library/src/include/hiptensor_options.hpp

```diff
@@ -49,16 +49,27 @@ namespace hiptensor
         HiptensorOptions(HiptensorOptions&&) = default;
         ~HiptensorOptions()                  = default;
 
-        void parseOptions(int argc, char** argv);
+        void setOstream(std::string file);
         void setOmits(int mask);
+        void setDefaultParams(bool val);
+        void setValidation(std::string val);
+        void setHotRuns(int runs);
+        void setColdRuns(int runs);
+        void setInputFilename(std::string file);
```

Comment:
We may want to be a bit more specific about what the files are. 
Suggest setInputYAMLFilename..
Suggest setOutputStreamFilename..